scripts/common/kubernetes.sh

```diff
@@ -92,6 +92,13 @@ function kubernetes_install_host_packages() {
 
     if kubernetes_host_commands_ok "$k8sVersion"; then
         logSuccess "Kubernetes host packages already installed"
+        # less command is broken if libtinfo.so.5 is missing in amazon linux 2
+        if [ "$LSB_DIST" == "amzn" ] && [ "$AIRGAP" != "1" ] && ! file_exists "/usr/lib64/libtinfo.so.5"; then
+            if [ -d "$DIR/packages/kubernetes/${k8sVersion}" ]; then
+                install_host_packages "${DIR}/packages/kubernetes/${k8sVersion}" ncurses-compat-libs
```

Comment:
will this fail if `ncurses-compat-libs` is not in the bundle?

Reply:
@emosbaugh 

No, it won't fail. I tested this on my codeserver and it was able to install the package even though it wasn't on the `build/packages/kubernetes/1.23.5/rhel-7-force` directory. 

I added the check for the directory because I saw this message on staging last week:
```
realpath: ‘./packages/kubernetes/1.23.5’: No such file or directory
An error occurred on line
```
when I ran it on an existing install. It may be because the packages weren't rebuilt but I'm hoping that this way the installer won't break if the package directory is not there.